roles/openstack_resource_controller/templates/kustomization.yaml.j2

```diff
@@ -5,3 +5,19 @@ images:
 - name: controller
   newName: {{ openstack_resource_controller_image }}
   newTag: {{ openstack_resource_controller_image_tag }}
+
+{% if openstack_resource_controller_node_selector %}
+patches:
```

Comment:
@mnaser Should we move the patches: outside the if block so that we can add other patches?